lib/IDE/ExprContextAnalysis.cpp

```diff
@@ -761,11 +763,6 @@ class ExprContextAnalyzer {
       if (!collectPossibleCalleesForSubscript(*DC, subscriptExpr, Candidates))
         return false;
       Arg = subscriptExpr->getIndex();
-    } else if (auto *unresolvedMemberExpr = dyn_cast<UnresolvedMemberExpr>(E)) {
-      if (!collectPossibleCalleesForUnresolvedMember(*DC, unresolvedMemberExpr,
-                                                     Candidates))
-        return false;
-      Arg = unresolvedMemberExpr->getArgument();
```

Comment:
You should be able to remove handing `UnresolvedMemberExpr` in `ExprContextAnalyzer::Analyze()`.
Here
https://github.com/apple/swift/blob/a350fb7f057aed8ef496eddcd86723413d8b6a97/lib/IDE/ExprContextAnalysis.cpp#L1147
and here
https://github.com/apple/swift/blob/a350fb7f057aed8ef496eddcd86723413d8b6a97/lib/IDE/ExprContextAnalysis.cpp#L1157